src/open_clip/factory.py

```diff
@@ -141,9 +142,11 @@ def create_model(
                 assert False, 'pretrained image towers currently only supported for timm models'
 
         cast_dtype = get_cast_dtype(precision)
-        custom_text = model_cfg.pop('custom_text', False) or force_custom_text or ('hf_model_name' in model_cfg['text_cfg'])
+        custom_text = model_cfg.pop('custom_text', False) or force_custom_text or ('hf_model_name' in model_cfg.get('text_cfg', {}))
```

Comment:
why this change

Reply:
because it could throw a KeyError if `text_cfg` is not in the config.
see https://github.com/mlfoundations/open_clip/blob/ea7718f927b84e1b46ce057d3eae5ca4c9c41434/src/open_clip/factory.py#L137
where it is done like this already.